packages/eslint-config-4catalyzer-react/test/config.test.js

```diff
@@ -1,5 +1,7 @@
 const pick = require('lodash/pick');
+
 const configTester = require('../../../tools/config-tester');
+
```

Comment:
what is the grouping here that this happens?

Reply:
I think parent imports are treated differently from sibling imports. not sure if it's a feature or a bug

Reply:
see the groups option